openslides_backend/action/actions/user/set_present.py

```diff
@@ -56,3 +59,33 @@ def get_updated_instances(self, action_data: ActionData) -> ActionData:
                     is_present.remove(meeting_id)
                     instance["is_present_in_meeting_ids"] = is_present
                     yield instance
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        meeting = self.datastore.get(
```

Comment:
You can fetch the meeting after checking the OML to avoid unnecessary datastore calls.

Reply:
Check the OML before getting the meeting.